src/main/java/datastructs/ISample.java

```diff
@@ -9,32 +9,33 @@
     /**
      * The name of the sample
      */
-    public String getName();
+    String name();
 
     /**
      * Returns the size of the sample
      */
-    public int getsize();
+    int size();
 
 	/**
 	 * Add the value to the sample
 	 */
-	public void add(T value);
+	void add(T value);
 
 	/**
 	 * Set the i-th entry to the given value
 	 */
```

Comment:
I don't quite get the documentation? could elaborate a little pls?

Reply:
Let's say the sample is ```[1, 2, 3, 4, 5]``` then ```set(2, 100)``` will cause ```[1, 2, 100, 4, 5]``` is that understood?

Reply:
now I get it